developers/retrievability.md

```diff
@@ -6,7 +6,7 @@ description: Practices and expectations for data retrievability and pruning on C
 # Data retrievability and pruning
 
 The purpose of data availability layers such as Celestia is to ensure
-that block data is provably published to the Internet, so that applications
+that block data is provably published, so that applications
```

Comment:
@evan-forbes i think leaving in "to the Internet" makes it clearer, but curious what you think.

Reply:
reviewed with Evan and good to go! thank you @howdymary